src/interfaces/save-content-node.interface.ts

```diff
@@ -1,5 +1,9 @@
+import {AssetMetadataTransport} from "../api/package-manager-api";
+
 export interface SaveContentNode {
+    id: string,
```

Comment:
I think this is not needed anymore as well, since we are using the new complete transports.